development/tests/test_match_details.py

```diff
@@ -130,3 +139,36 @@ def test_should_shows_pass_data_to_template_when_it_is_received_from_server_in_n
             response.context_data['data'],
             "testing"
         )
+
+    @parameterized.expand([
+        ([INVALID_STATE, VALID_STATE, ALL_STATES], [ALL_ACTIONS],),
+        ([VALID_STATE, ALL_STATES], [MOVE_ACTION, SHOOT_ACTION, ALL_ACTIONS],),
+    ])
+    @patch('development.views.get_logs')
+    @patch('development.views.generate_text')
+    @patch.object(FilterLogs, 'possible_states', new_callable=PropertyMock)
+    @patch.object(FilterLogs, 'possible_actions', new_callable=PropertyMock)
+    def test_should_shows_filter_logs_options_to_template_when_it_is_received_from_server(
+        self,
+        possible_states,
+        possible_actions,
+
```

Comment:
```suggestion
```

Reply:
I live in purpose that line to difference with the other variables. Is it not a good practice? I'll change it in any case